.github/workflows/R-CMD-check.yaml

```diff
@@ -34,9 +34,6 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@v2
       
-      - name: Install curl
```

Comment:
This makes Windows CI fail. It seems like it is an unnecessary library for ubuntu CI, probably a remnant from Jenkins.